.github/workflows/installation-cli.yaml

```diff
@@ -36,32 +49,28 @@ jobs:
         uses: actions/setup-go@v6
         with:
           go-version-file: go.mod
-      - name: run karmadactl init test
+      - name: setup init e2e test environment
         run: |
           export CLUSTER_VERSION=kindest/node:${{ matrix.k8s }}
-
-          # init e2e environment
-          hack/cli-testing-environment.sh
-
-          # run a single e2e
-          export PULL_BASED_CLUSTERS="member1:${HOME}/.kube/member1.config"  
-          export KUBECONFIG=${HOME}/.kube/karmada-host.config:${HOME}/karmada/karmada-apiserver.config
-          GO111MODULE=on go install github.com/onsi/ginkgo/v2/ginkgo
-          ginkgo -v --race --trace -p  --focus="[BasicPropagation] propagation testing deployment propagation testing"  ./test/e2e/suites/base
-      - name: export logs
-        if: always()
+          hack/init-e2e-environment.sh
+      - name: run e2e
         run: |
-          export ARTIFACTS_PATH=${{ github.workspace }}/karmadactl-test-logs/${{ matrix.k8s }}/
-          mkdir -p $ARTIFACTS_PATH
-
-          mkdir -p $ARTIFACTS_PATH/karmada-host
-          kind export logs --name=karmada-host $ARTIFACTS_PATH/karmada-host
+          export ARTIFACTS_PATH=${{ github.workspace }}/karmada-init-e2e-logs/${{ matrix.k8s }}/
+          export REGISTRY="docker.io/karmada"
+          export VERSION="latest"
+          hack/run-e2e-init.sh
       - name: upload logs
         if: always()
         uses: actions/upload-artifact@v6
         with:
-          name: karmadactl_test_logs_${{ matrix.k8s }}
-          path: ${{ github.workspace }}/karmadactl-test-logs/${{ matrix.k8s }}/
+          name: karmada_init_e2e_log_${{ matrix.k8s }}
+          path: ${{ github.workspace }}/karmada-init-e2e-logs/${{ matrix.k8s }}/
+      - name: upload kind logs
+        if: always()
+        uses: actions/upload-artifact@v5
```

Comment:
I noticed that the version of the tool used in the ci.yml file is v6. Should this be consistent here?